features/tailormade_face_swap/src/main/java/com/future/tailormade_face_swap/util/BitmapHelper.kt

```diff
@@ -0,0 +1,49 @@
+package com.future.tailormade_face_swap.util
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.graphics.Canvas
+import android.graphics.Matrix
+import com.bumptech.glide.Glide
+import java.io.File
+
+object BitmapHelper {
+
+  fun convertByteArrayToBitmap(byteArray: ByteArray): Bitmap = BitmapFactory.decodeByteArray(
+      byteArray, 0, byteArray.size)
+
+  fun convertFilePathToBitmap(filePath: String): Bitmap? = if (File(filePath).exists()) {
+    BitmapFactory.decodeFile(filePath)
+  } else {
+    null
+  }
+
+  fun convertUrlToBitmap(context: Context, imageUrl: String): Bitmap = Glide.with(context).asBitmap().load(
+      imageUrl).submit().get()
+
+  fun getAdjustedBitmapSize(bitmapDestination: Bitmap, bitmapSource: Bitmap): Pair<Bitmap, Bitmap> {
+    val maxWidth = maxOf(bitmapDestination.width, bitmapSource.width)
+    val maxHeight = maxOf(bitmapDestination.height, bitmapSource.height)
+
+    val overlayBitmap1 = getOverlayImage(maxWidth, maxHeight, bitmapDestination)
```

Comment:
overlayBitmap1 can be overlayBitmapDestination
overlayBitmap2 cab be overlatBitmapSource